README.md

```diff
@@ -3,7 +3,3 @@
 This repo contains work-in-progress documentation for .NET. To contribute, see the [Contributing Guide](https://github.com/dotnet/docs/blob/master/CONTRIBUTING.md) and the [issues list](https://github.com/dotnet/docs/issues). 
```

Comment:
This should be modified to reflect its actual contents. as in contributing.md. Instead of just the link to issues, also note that to see samples issues, filter by the Repo - Samples label.